packages/react-instantsearch/src/core/createInstantSearchManager.error.test.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-env jest, jasmine */
+/* eslint-disable no-console */
+
+import createInstantSearchManager from './createInstantSearchManager';
+
+import algoliaClient from 'algoliasearch';
+
+jest.useFakeTimers();
+
+jest.mock('algoliasearch-helper/src/algoliasearch.helper.js', () => {
+  let count = 0;
+  console.log('setup');
+  const Helper = require.requireActual('algoliasearch-helper/src/algoliasearch.helper.js');
+  Helper.prototype._handleResponse = function(state) {
+    this.emit('error', {count: count++}, state);
```

Comment:
To avoid having several files, maybe something you could do is emitting a different event depending of the query present in the state parameter?